### PR TITLE
release: v7.0.0

### PR DESCRIPTION
**Pull request by snprajwal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ The following table shows the relation between go-criu and criu versions:
 
 | Major version  | Latest release | CRIU version |
 | -------------- | -------------- | ------------ |
+| v7             | 7.0.0          | 3.18         |
```
Comment (rst0git) [inline README.md:65]: @snprajwal There are more files that need to be updated:
```bash
$ ag -c go-criu/v6
phaul/client.go:4
phaul/api.go:1
phaul/server.go:2
test/phaul/main.go:3
test/crit/main.go:3
test/main.go:2
crit/images/bpfmap-file/bpfmap-file.pb.go:2
crit/images/core-aarch64/core-aarch64.pb.go:1
crit/images/core-arm/core-arm.pb.go:1
crit/images/core-mips/core-mips.pb.go:1
crit/images/core-ppc64/core-ppc64.pb.go:1
crit/images/core-s390/core-s390.pb.go:1
crit/images/core-x86/core-x86.pb.go:1
crit/images/criu-core/criu-core.pb.go:13
crit/images/criu-sa/criu-sa.pb.go:1
crit/images/eventfd/eventfd.pb.go:1
crit/images/eventpoll/eventpoll.pb.go:1
crit/images/ext-file/ext-file.pb.go:1
crit/images/fdinfo/fdinfo.pb.go:18
crit/images/fh/fh.pb.go:1
crit/images/fsnotify/fsnotify.pb.go:3
crit/images/ghost-file/ghost-file.pb.go:2
crit/images/inventory/inventory.pb.go:1
crit/images/ipc-sem/ipc-sem.pb.go:1
crit/images/ipc-shm/ipc-shm.pb.go:1
crit/images/memfd/memfd.pb.go:2
crit/images/mm/mm.pb.go:2
crit/images/mnt/mnt.pb.go:1
crit/images/netdev/netdev.pb.go:5
crit/images/packet-sock/packet-sock.pb.go:3
crit/images/pagemap/pagemap.pb.go:1
crit/images/pipe/pipe.pb.go:2
crit/images/regfile/regfile.pb.go:2
crit/images/ipc-msg/ipc-msg.pb.go:1
crit/images/sit/sit.pb.go:1
crit/images/signalfd/signalfd.pb.go:2
crit/images/sk-unix/sk-unix.pb.go:3
crit/images/sk-netlink/sk-netlink.pb.go:3
crit/images/sk-inet/sk-inet.pb.go:3
crit/images/tty/tty.pb.go:2
crit/images/timerfd/timerfd.pb.go:2
crit/images/tcp-stream/tcp-stream.pb.go:1
crit/images/vma/vma.pb.go:1
crit/cmd/main.go:1
crit/encode-extra.go:1
crit/encode.go:1
crit/image.go:2
crit/stats.go:1
crit/decode.go:2
crit/explore.go:6
crit/utils.go:5
crit/mempages.go:2
crit/cli/handler.go:54
crit/cli/cli.go:1
crit/images/tun/tun.pb.go:1
crit/decode-extra.go:7
scripts/proto-gen/protogen.py:1
crit/mempages_test.go:1
main.go:1
features.go:1
README.md:1
```

File: crit/utils.go [hunk 2]
```diff
@@ -267,9 +267,9 @@ func processIP(parts []uint32) string {
 		binary.LittleEndian.PutUint32(ip, parts[0])
 		return ip.String()
 	}
-	// IPv6
+	// IPv7
```
Comment (rst0git) [inline crit/utils.go:270]: this should be `IPv6`

File: crit/utils.go [hunk 3]
```diff
@@ -267,9 +267,9 @@ func processIP(parts []uint32) string {
...
 	if len(parts) == 4 {
-		ip := make(net.IP, net.IPv6len)
+		ip := make(net.IP, net.IPv7len)
```
Comment (rst0git) [inline crit/utils.go:272]: ditto

File: crit/utils.go (hunk 2 above)
Comment (snprajwal) [reply crit/utils.go:270]: Oops, I the sed command changed this too :skull: I'm sorting it out

**Review by rst0git — APPROVED** [review]

**Pull request merged**